uk.co.compucorp.civicrm.hremails/email-templates/src/pages/drupal-account-activated.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: civihr-default
+title:  Welcome to CiviHR!
+---
+
+<container>
+
+  <row>
+    <columns bgcolor="#ffffff">
+      <spacer></spacer>
+      <h2>Please create a password</h2>
+      <spacer></spacer>
+      Hello, and welcome to CiviHR, the open-source HR solution for non-profit organisations.
```

Comment:
shouldn't the text be wrapped in a `<p>` *or whatever equivalent custom tag foundation gives us) instead of having to add a spacer every time?

Reply:
I was experimenting with `<p>` when I found the problem with Inbox. The end result is just whatever seemed to be looking good at the time, but I'll try using `<p>`. If the default spacing with `<p>` isn't enough do you think it would be better to add a class to that `<p>` and adjust the spacing with CSS or use `<p>` in combination with `<spacer>`?

Reply:
@mickadoo better to use `<p>` only, with maybe a class. Having to remember to use `<p>` + `<spacer>` every time is just annoying

Reply:
This shouldn't be an issue now since the markup is coming from Drupal